examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -19,6 +19,9 @@ class MainApplication : Application(), UpdatedCustomerInfoListener {
 
     val logHandler = TesterLogHandler(this)
 
+    private val customerInfoListeners: MutableSet<UpdatedCustomerInfoListener> = mutableSetOf()
+    private var lastCustomerInfo: CustomerInfo? = null
```

Comment:
I thought about moving the listener to exist in the fragment itself. However, we might need this info in other screens in the future, so leaving it in the application for now. 

If we were following proper android architecture, it should really be on a repository class behind the scenes... But don't want to enter into that rabbit hole right now.